tests/api_send/test_send_super.py

```diff
@@ -74,7 +74,7 @@ def setUpClass(cls):
                 "init_supply": 10000
             },
             # It is used to send message only.
-            "data": "test"
+            "data": "0x" + "test".encode().hex()
```

Comment:
I think that

before : encode()
after : encode(utf-8)

Reply:
But does not it work as same? Default is utf-8.